notification/src/main/resources/application-test.yml

```
@@ -0,0 +1,20 @@
+    hibernate:
+      ddl-auto: create
+    properties:
+      hibernate:
+        spring.jpa.open-in-view: false
+        format_sql: true
+        default_batch_fetch_size: 10
+
+eureka:
+  client:
+    register-with-eureka: false  # Eureka 서버에 등록
+    fetch-registry: false  # Eureka 서버로부터 레지스트리 정보 가져오기
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)